plugins/follow_me/follow_me.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include <iostream>
+#include <cmath>
+
+namespace dronecore {
+
+class FollowMeImpl;
+
+class FollowMe
+{
+public:
+    explicit FollowMe(FollowMeImpl *impl);
+    ~FollowMe();
+
+    /**
+     * @brief Results of FollowMe operations
+     */
+    enum class Result {
+        SUCCESS = 0, /**< @brief %Request succeeded. */
+        NO_DEVICE, /**< @brief No device connected. */
+        CONNECTION_ERROR, /**< @brief %Connection error. */
+        BUSY, /**< @brief Vehicle busy. */
+        COMMAND_DENIED, /**< @brief Command denied. */
+        TIMEOUT, /**< @brief %Request timeout. */
+        UNKNOWN /**< @brief Unknown error. */
+    };
+
+    /**
+     * @brief Altitude flag for Follow Target command.
+     */
+    enum class ESTCapabilities {
+        POS = 0,
+        VEL = 1,
+        ACCEL = 2,
+        ATT_RATES = 3
+    };
+
+    /**
+     * @brief Motion report info which will be emitted peridically to the Vehicle
+     */
+    struct MotionReport {
+        // set to home position used by PX4.
+        int32_t lat_int = static_cast<uint32_t>(47.3977419 *
```

Comment:
if this is home position, name it appropriately => home_latitude. No need to put "type" in name. If this is public API then have a doc string output with @brief tag.

Reply:
It is not public API now.